sickbeard/webserve.py

```diff
@@ -1634,6 +1634,19 @@ def titler(x):
             'name': showObj.name,
         })
 
+        episode_status = None
+        release_name = None
+        try:
+            main_db_con = db.DBConnection()
+            episode_status_result = main_db_con.action("SELECT action, resource from history where showid = ? and season = ? and episode = ? order by date DESC LIMIT 1 ",  
+                                                             [indexerid, season, episode])
+            if episode_status_result:
+                for item in episode_status_result:
+                    episode_status = int(item['action'])
```

Comment:
I had to do a loop because doing episode_status_result[0]['action'] didn't worked